index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Tribute Page</title/>
+    <title>Tribute-Page</title/>
```

Comment:
Great job adding the inline review. 
Kindly state the reason why it is good to follow semantic tags. This will make the programmer know the reason for making the changes. ☺️ 🟢 
- For easy reception kindly state your suggestions politely ☺️ 📍 
- I think it will be good if you make use of emojis and gifs. This will make your code review more human-friendly 👽🤖 ⚖️